src/lib/helpers/ArrowHelper.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+  import type { ArrowHelperProperties, Object3DProperties } from '$lib/types/components';
+  import { ArrowHelper as ThreeArrowHelper } from 'three'
+  import Object3DInstance from '../instances/Object3DInstance.svelte'
+
+  // Object3DInstance
+  export let position: Object3DProperties['position'] = undefined
+  export let scale: Object3DProperties['scale'] = undefined
+  export let rotation: Object3DProperties['rotation'] = undefined
+  export let lookAt: Object3DProperties['lookAt'] = undefined
+  export let viewportAware: Object3DProperties['viewportAware'] = false
+  export let inViewport: Object3DProperties['inViewport'] = false
+  export let castShadow: Object3DProperties['castShadow'] = undefined
+  export let receiveShadow: Object3DProperties['receiveShadow'] = undefined
+  export let frustumCulled: Object3DProperties['frustumCulled'] = undefined
+  export let renderOrder: Object3DProperties['renderOrder'] = undefined
+  export let visible: Object3DProperties['visible'] = undefined
+  
+  //self
+  export let dir:ArrowHelperProperties['dir'] = undefined;
+  export let origin:ArrowHelperProperties['origin'] = undefined;
+  export let length:ArrowHelperProperties['length'] = undefined;
+  export let color:ArrowHelperProperties['color'] = undefined;
+  export let headLength:ArrowHelperProperties['headLength'] = undefined;
+  export let headWidth:ArrowHelperProperties['headWidth'] = undefined;
+
+  let helper: ThreeArrowHelper = new ThreeArrowHelper (dir,origin,length,color,headLength,headWidth);
```

Comment:
helper is never reassigned: use `const` and export it so that users can bind onto it: `export const arrowHelper = new ThreeArrowHelper(…)`.